src/browser/main.coffee

```diff
@@ -56,11 +56,24 @@ handleStartupEventWithSquirrel = ->
 setupCrashReporter = ->
   crashReporter.start(productName: 'Atom', companyName: 'GitHub')
 
-setupAtomHome = ->
+isPortableInstall = (args) ->
+  return false unless process.platform is 'win32'
+  return false unless (process and process.type)
+  return false if args.devMode or args.testMode
+
+  ourPath = process.execPath.toLowerCase()
```

Comment:
Could you add a comment above this line that just adds some example paths and says which ones are considered portable and which ones are not?